src/js/components/ServiceOptions.js

```diff
@@ -0,0 +1,115 @@
+import classNames from 'classnames';
+import mixin from 'reactjs-mixin';
+import React from 'react';
+import {StoreMixin} from 'mesosphere-shared-reactjs';
+
+import CosmosPackagesStore from '../stores/CosmosPackagesStore';
+import Framework from '../structs/Framework';
+import InternalStorageMixin from '../mixins/InternalStorageMixin';
+import Service from '../structs/Service';
+
+const METHODS_TO_BIND = [
+  'handleEditConfigClick',
+  'handleConfigModalClose'
+];
+
+class ServiceOptions extends mixin(InternalStorageMixin, StoreMixin) {
+  constructor() {
+    super(...arguments);
+
+    this.state = {
+      editConfigModalOpen: false
+    };
+
+    this.internalStorage_set({
+      servicePackage: null
```

Comment:
That's a good point, but since `CosmosPackagesStore` only stores `packageDetails` for one package at a time, I don't know for sure if it's the package I want.

With your suggestion, it's possible that `CosmosPackagesStore#getPackageDetails` will return details for a package that I previously had selected.

With the way I have it implemented currently, I know for sure that the `packageDetails` will pertain to the service that I just opened, because I assume it's null until the fetch is successful.

What do you think? Perhaps we should change the way `CosmosPackagesStore` stores and retrieves package details?